public/lusc_cptac_2021/meta_methylation_epic.txt

```diff
@@ -0,0 +1,8 @@
+cancer_study_identifier: lusc_cptac_2021
+genetic_alteration_type: METHYLATION
+datatype: CONTINUOUS
+stable_id: methylation_epic
```

Comment:
stable ID in methylation is a limited namespace. Validator throws an error:
```
ERROR: meta_methylation_epic.txt: Invalid stable id for genetic_alteration_type 'METHYLATION', data_type 'CONTINUOUS'; expected one of [methylation_hm27, methylation_hm450]; value encountered: 'methylation_epic'
```

Reply:
Hi,
I have kept methylation_epic as stable id, because that is the one that @yichaoS used in a previous study ([luad_cptac_2020](https://github.com/cBioPortal/datahub/pull/1503)), so I guessed that this stable id will pass validation in cBioPortal on their side?

I changed the stable id lusc_cptac_2021_custom_all to lusc_cptac_2021_all to avoid the second error message.

Reply:
@yichaoS can you verify what Elena says here? It it OK to use this stable ID?

Reply:
@SRodenburg @egarcialara Yes, we created this new profile in portal and the stable ID is `methylation_epic`. (for methylations done on the EPIC platform). I've changed the methylation profile meta in other CPTAC studies to this as well.

Reply:
OK I guess then the validator needs an update :)

Reply:
@SRodenburg I've updated the `allowed_data_type` sheet with the new data types in the cbioportal repo.